optimum/commands/export/openvino.py

```diff
@@ -139,5 +144,6 @@ def run(self):
             pad_token_id=self.args.pad_token_id,
             compression_option=self.args.weight_format,
             compression_ratio=self.args.ratio
```

Comment:
```suggestion
            compression_ratio=self.args.ratio,
```

Reply:
fixed in https://github.com/huggingface/optimum-intel/pull/493